Client/Frontend/Browser/Playlist/Controllers/PlaylistListViewController.swift

```diff
@@ -247,20 +255,37 @@ class PlaylistListViewController: UIViewController {
     }
     
     // Shared folders are loading
-    if PlaylistManager.shared.currentFolder == nil {
+    if PlaylistManager.shared.currentFolder?.isPersistent == false {
```

Comment:
Shared folders that are NOT persistent (not stored on the device, but only shown via the website) should not autoplay. That's why :)

This also checks existence because `currentFolder` is optional